### DISCUSSION TITLE
Vercel can't resovle correct import path.

### DISCUSSION THREAD

**Discussion by noefoy** [vercel/next.js#68702] [Help]
### Summary

Guys am I missing something here I keep trying to deploy my React Site via Vercel but I keep getting this error: 

```
error during build:
Could not resolve "../components/TimelineItem" from "src/pages/education.jsx"
file: /vercel/path0/src/pages/education.jsx
    at getRollupError (file:///vercel/path0/node_modules/rollup/dist/es/shared/parseAst.js:396:41)
    at error (file:///vercel/path0/node_modules/rollup/dist/es/shared/parseAst.js:392:42)
    at ModuleLoader.handleInvalidResolvedId (file:///vercel/path0/node_modules/rollup/dist/es/shared/node-entry.js:19102:24)
    at file:///vercel/path0/node_modules/rollup/dist/es/shared/node-entry.js:19062:26
Error: Command "npm run build" exited with 1
```
I double-checked the path and the casing a million times and I'm sure its correct, but I keep getting this same error...

<img width="847" alt="Screenshot 2024-08-09 at 04 54 01" src="https://github.com/user-attachments/assets/347850e7-cdb4-433c-93cb-3611f487ca7d">

<img width="588" alt="Screenshot 2024-08-09 at 04 53 50" src="https://github.com/user-attachments/assets/c7b00f5b-74b2-49e0-8428-3351887a6b43">

My directory:
<img width="235" alt="Screenshot 2024-08-09 at 04 53 34" src="https://github.com/user-attachments/assets/9639b7f3-b0eb-4af9-a717-bb7a91da5a07">

**Comment by Kareem21** [comment]
Same shit

**Reply by metadol** [reply]
did u find a fix?

**Reply by Kyaee** [reply]
same situation :((

**Reply by vigiperez24** [reply]
same s***
